doc/src/manual/methods.md

```diff
@@ -263,10 +263,7 @@ Possible fix, define
   g(::Float64, ::Float64)
 ```
 
-Here the call `g(2.0, 3.0)` could be handled by either the `g(Float64, Any)` or the `g(Any, Float64)`
-method, and neither is more specific than the other. In such cases, Julia raises a [`MethodError`](@ref)
-rather than arbitrarily picking a method. You can avoid method ambiguities by specifying an appropriate
-method for the intersection case:
+The order in which the methods are defined does not matter and neither is more specific than the other.  Here the call `g(2.0, 3.0)` could be handled by either the `g(Float64, Any)` or the `g(Any, Float64)` method. In such cases, Julia raises a [`MethodError`](@ref) rather than arbitrarily picking a method. You can avoid method ambiguities by specifying an appropriate method for the intersection case:
```

Comment:
Can you split this up into multiple lines? We generally try to adhere to a maximum line length of 92 characters, because it often makes diffs easier to read and is easier to edit.

Reply:
Sorry Simeon, this is my very first PR. Do you mean to split the paragraph into two parts? Or do you mean that I should add a line break at the end of each sentence? Thanks!

Reply:
Yes, I meant just adding some newlines in between, doesn't have to only be between sentences. Something like:
```suggestion
The order in which the methods are defined does not matter and neither is more specific
than the other. Here the call `g(2.0, 3.0)` could be handled by either the
`g(Float64, Any)` or the `g(Any, Float64)` method. In such cases, Julia raises a
[`MethodError`](@ref) rather than arbitrarily picking a method. You can avoid method
ambiguities by specifying an appropriate method for the intersection case: 
```